packages/polaris-schema/src/scalars/polaris-scalars-resolvers.ts

```diff
@@ -33,9 +33,16 @@ import {
 } from 'graphql-scalars';
 import { IResolvers } from 'graphql-tools';
 
-export const scalarsResolvers: IResolvers = {
+export const getScalarsResolvers = (shouldAddGraphQLScalars: boolean): IResolvers => {
+    return shouldAddGraphQLScalars ? polarisScalarsResolvers : defaultPolarisScalarsResolvers;
+};
+
+export const defaultPolarisScalarsResolvers: IResolvers = {
```

Comment:
where is update

Reply:
only type def is needed. it is given by apollo server